src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -828,6 +828,7 @@ def load_arguments(self, _):
         c.argument('enable_vtpm', enable_vtpm_type)
         c.argument('custom_data', help='Custom init script file or text (cloud-init, cloud-config, etc..)', completer=FilesCompleter(), type=file_type)
         c.argument('security_type', arg_type=get_enum_type(["TrustedLaunch"]), min_api='2020-06-01', help='Specify the security type of the virtual machine scale set.')
+        c.argument('security_type', arg_type=get_enum_type(["TrustedLaunch", "Standard"]), min_api='2023-07-01', help='Specify the security type of the virtual machine scale set.')
```

Comment:
Just out of curiosity, is `Standard` only supported after api version `2023-07-01`?

Reply:
correct, it is mentioned in the [feature request](https://github.com/Azure/azure-cli/issues/28991#issue-2305127576):
![image](https://github.com/Azure/azure-cli/assets/105691024/814a20d2-cef3-47d5-9476-f7a2acb1a2c1)

Reply:
> This looks a bit strange
> 
> > Just out of curiosity, is Standard only supported after api version 2023-07-01?
> 
> @AjKundnani Could you please confirm this question?

@zhoxing-ms - Thanks for catching this, min API version for roll-back support is `2020-06-01`. Will update the request as well.
cc @yanzhudd